assets/stylesheets/sass/main/main.scss

```diff
@@ -609,52 +623,27 @@ figure {
   }
 
   .button{
-    text-transform:uppercase;
-    background-color:lighten($brand-red, 1);
+    background-color: lighten($brand-red, 1);
+    padding: 1.1em 1.4em;
+    text-transform: uppercase;
+
     &:hover{
-      background-color:lighten($brand-red-darker,5 );
+      background-color: lighten($brand-red-darker,5 );
```

Comment:
Commas in function arguments should be followed by a single space<br>Expected 0 spaces between parentheses instead of 1